pkg/util/util.go

```diff
@@ -1238,6 +1238,20 @@ func GetFlagStringOrPromptPassword(cmd *cobra.Command, flag string) string {
 	return str
 }
 
+// GetBoolFlagPtr returns a pointer to the boolean flag value if set, or nil if not set
+func GetBoolFlagPtr(cmd *cobra.Command, flag string) (*bool, error) {
+	flagVal, err := cmd.Flags().GetBool(flag)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
If I am adding this condition, the admission-tests are failing with the below error:
`time="2024-09-20T12:55:53Z" level=fatal msg="flag accessed but not defined: force_md5_etag"`

I think we need not to handle this condition😅 just like other functions in util.go, also without it the tests are passing. As we are returning nil value from the function if the flag is not found.
@dannyzaken  wdyt?

Reply:
What is the root cause of this error? I am not sure ignoring the error is the answer